lib/logstash/filters/geoip.rb

```diff
@@ -124,22 +124,24 @@ def register
 
   public
   def filter(event)
-    
+
     geo_data = nil
 
     geo_data = get_geo_data(event)
 
-    return if geo_data.nil? || !geo_data.respond_to?(:to_hash)
-
-    apply_geodata(geo_data, event)
+    return unless geo_data.respond_to?(:to_hash)
```

Comment:
from my understanding this will never be executed since we now control the output of get_geo_data and either pump out an empty hash or a geoip struct?